Sources/ContainerClient/Core/Volume.swift

```diff
@@ -58,6 +58,16 @@ public struct Volume: Sendable, Codable, Equatable, Identifiable {
     }
 }
 
+extension Volume {
+    /// Reserved label key for marking anonymous volumes
+    public static let anonymousLabel = "com.apple.container.volume.anonymous"
```

Comment:
Let's discuss using either `com.apple.container.resource.role=anonymous`, or `com.apple.container.resource.anonymous` like you're using it now.